fashion-messaging-app-api/server.js

```diff
@@ -173,45 +172,103 @@ app.get('/recommendations', async (req, res) => {
       return res.status(401).json({ error: 'Unauthorized' });
     }
 
+    let recommendations = [];  
+    const frequentSearch = await SearchResult.findOne({
+      where: { 
+        userId: req.session.user.id,
+        searchTerm: {
+          [Op.not]: defaultQuery  
+        }
+      },
+      order: [['searchCount', 'DESC']],
+    });
     const lastSearch = await SearchResult.findOne({
-      where: { userId: req.session.user.id },
+      where: { 
+        userId: req.session.user.id,
+        searchTerm: {
+          [Op.not]: defaultQuery 
+        }
+      },
       order: [['createdAt', 'DESC']],
     });
 
-    const mostFrequentSearch = await SearchResult.findOne({
-      where: { userId: req.session.user.id },
-      order: [['searchCount', 'DESC']],
+    const popularSearch = await SearchResult.findOne({
```

Comment:
Minor comment- The naming of frequentSearch and popularSearch confuse me because I feel like those are the same thing

Reply:
Can you explain more about the difference between what frequentSearch and popularSearch are? I'm wondering why the queries are so different